lib/Conversion/TorchToLinalg/Random.cpp

```diff
@@ -12,6 +12,7 @@
 #include "PopulatePatterns.h"
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/Linalg/IR/Linalg.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
```

Comment:
If I understand correctly, it is not acceptable to use scf dialect as a target for a torchtolinalg pass. I think many of your scf loops can be re-expressed as linalg generic ops, so please try to rewrite this pass without scf (and change to camelCase as well).

Reply:
Got it, I'll try to refactor as linalg generics instead